packages/js/src/plugins/auctionHouseModule/operations/createListing.ts

```diff
@@ -407,6 +407,7 @@ export const createListingBuilder = (
     key.pubkey.equals(signer.publicKey)
   );
   sellInstruction.keys[signerKeyIndex].isSigner = true;
+  sellInstruction.keys[signerKeyIndex].isWritable = true;
```

Comment:
Are you sure it needs to be writable?

Reply:
I remember I had an issue when I was using an external seller. Will check again and add a comment.